content/reference/plugins/grafana/index.md

```diff
@@ -111,7 +111,7 @@ Gatling Enterprise offers a large amount of metrics:
     - `usr+`: Users arrival count per second.
     - `usr-`: Users termination count per second.
     - `usrActive`: Concurrent users count per second.
-- **Injectors metrics:**
+- **Load Generators metrics:**
     - `gcCount`: GC count per second.
```

Comment:
`user/dedicated_ips/dedicated-ips.png`: table is empty

Reply:
@ccedric What do you want? Asking for dedicated_ips for this account or using the demo account?